### PR TITLE
add readme*.md nodemon.* pm2.* file

### PR DESCRIPTION
**Pull request by hubvue**
**Some explanations**

readme*.md: [add i18n readme*.md, for example: readme.zh_CN.md](https://github.com/antfu/vscode-file-nesting-config/commit/3555b76aaa58b3d1d67c090979124e1c45cbbd98)
.air*: air is to go what nodemon is to node

### CODE REVIEW

File: update.mjs [hunk 1]
```diff
@@ -145,6 +147,7 @@ const readme = [
   'contributors',
   'maintainers',
   'credits',
+  'readme*.md'
```
Comment (antfu) [inline update.mjs:150]: This has already been covered by the first item

File: update.mjs [hunk 2]
```diff
@@ -6,6 +6,8 @@ const buildTools = [
   'rollup.config.*',
   'webpack.config.*',
   'gulp*',
+  'nodemon.*',
```
Comment (antfu) [inline update.mjs:9]: We have this already

File: update.mjs [hunk 3]
```diff
@@ -6,6 +6,8 @@ const buildTools = [
...
+  'pm2.*'
```
Comment (antfu) [inline update.mjs:10]: Let's move pm2 to packageJSON list

File: update.mjs (hunk 1 above)
Comment (hubvue) [reply update.mjs]: Yes, I also just found out after pull request

File: update.mjs (hunk 2 above)
Comment (hubvue) [reply update.mjs]: nodemon* is also available for configuration, for example: nodemon.json

**Pull request merged**